### PR TITLE
[twRO] support

### PR DESCRIPTION
**Pull request by sctnightcore**

### CODE REVIEW

File: tables/twRO/recvpackets.txt [hunk 1]
```diff
@@ -1,1548 +1,1583 @@
...
+09FE -1 83 1
+09FF -1 84 1
+0A00 269 269 1
+0A01 3 3 0
+0A02 4 4 1
+0A09 55 55 1
+0A0A 57 57 1
+0A0B 57 57 1
+0A0C 66 66 1
+8B00000A -1 4 1
```
Comment (alisonrag) [inline tables/twRO/recvpackets.txt:1010]: need to check this packet

Comment (sctnightcore) [reply tables/twRO/recvpackets.txt]: fixed

**Review by alisonrag — APPROVED** [review]

File: tables/twRO/recvpackets.txt [hunk 2]
```diff
@@ -1,1548 +1,1583 @@
...
-0A04 6
-0A05 63
-0A06 6
-0A07 9
-0A08 26
-0A09 55
-0A0A 57
-0A0B 57
-0A0C 66
-0A0D -1
```
Comment (ya4ept) [inline tables/twRO/recvpackets.txt:1200]: `inventory_items_nonstackable 0A0D`
we need to check this package. It is missing from the new version of the file

**Review by ya4ept — CHANGES_REQUESTED** [review]

File: tables/twRO/recvpackets.txt (hunk 1 above)
Comment (ya4ept) [reply tables/twRO/recvpackets.txt]: @alisonrag do we only need to check the last packet or all three?

File: control/config.txt [hunk 3]
```diff
@@ -839,4 +839,4 @@ logToFile_Errors
 logToFile_Messages
 logToFile_Warnings
 history_max 50
-gameGuard 0
+gameGuard 1
```
Comment (alisonrag) [inline control/config.txt:842]: this should not be here

Comment (ya4ept) [reply control/config.txt:842]: We will remove this when we merge with the master. I am planning to remove this setting from config.txt

**Review by ya4ept — APPROVED** [review]

**Pull request merged**